Dockerfile

```diff
@@ -17,5 +17,8 @@ COPY src ./src
 # Copy Data Access Layer project sources
 COPY dal ./dal
 
+# Convert gradlew file to unix format
+RUN ["dos2unix", "./gradlew"]
```

Comment:
If I recall correctly we agreed on solution using Unix Line Endings (LF) for such cases.
This solution works but looks like a 'hack'.

In cases we have multiple shell scripts for linux this approach could look ugly, running `dos2unix` for each script.
My proposal is to always use LF for git repos and document this.

What do you think?

Reply:
Indeed, that was the intention. We only strictly require unix line endings in the executable files which we only have one at the moment.

Reply:
let's discuss this all together in one room?